### ISSUE TITLE
Reduce log noise

### ISSUE THREAD

**Issue by tomalrussell** [tomalrussell/colouring-london#216]
The following error is expected when a user hasn't yet liked a building:

```
QueryResultError {
    code: queryResultErrorCode.noData
    message: "No data returned from the query."
    received: 0
    query: "SELECT true as like FROM building_user_likes WHERE building_id = '3114522' and user_id = '123' LIMIT 1"
}
```

Rewrite the query or suppress the log message when handling.

**Comment by tomalrussell** [comment]
Here's the function: https://github.com/tomalrussell/colouring-london/blob/master/app/src/api/building.js#L86-L96

`db` is a [`pg-promise`](http://vitaly-t.github.io/pg-promise/index.html) database connection.

Fix might be a matter of switching from `db.one` to [`db.oneOrNone`](http://vitaly-t.github.io/pg-promise/Database.html#oneOrNone)

**Comment by martin-dj** [comment]
Submitted fix to feature branch. I think the issue is resolved now.

**Comment by tomalrussell** [comment]
Thanks @martin-dj - if you want to open pull requests for each branch that's ready to merge, I'll review and merge in at some point over the next couple of days.

**Comment by tomalrussell** [comment]
Closed by #224
